styles.css

```diff
@@ -0,0 +1,83 @@
+body {
```

Comment:
-great structure in CSS. However, I would like to kindly suggest adding linters to your projects as A linter is great for identifying errors when you use standard rules, it would make your code automated by analyzing your source code to look for bugs and stylistic errors.